Student/StudentUnitTests/Assignments/AssignmentDetails/AssignmentDetailsViewControllerTests.swift

```diff
@@ -210,7 +210,7 @@ class AssignmentDetailsViewControllerTests: StudentTestCase {
         XCTAssertEqual(viewController.gradeSection?.isHidden, false)
         XCTAssertEqual(viewController.gradeCellDivider?.isHidden, false)
         XCTAssertEqual(viewController.gradedView?.isHidden, true)
-        XCTAssertEqual(viewController.submittedView?.isHidden, true)
+        XCTAssertEqual(viewController.submittedView?.isHidden, false)
```

Comment:
This is in `testUpdateGradeCellWhenThereIsUploadState`. If there is an upload state we need to be showing the `submittedView`.